static/js/components/NewAllocation.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Form from "@rjsf/material-ui";
+import { showNotification } from "baselayer/components/Notifications";
+import { submitAllocation } from "../ducks/allocation";
+import { fetchAllocations } from "../ducks/allocations";
+
+const NewAllocation = () => {
+  const { instrumentList } = useSelector((state) => state.instruments);
+  const { telescopeList } = useSelector((state) => state.telescopes);
+  const groups = useSelector((state) => state.groups.userAccessible);
+  const dispatch = useDispatch();
+
+  const handleSubmit = async ({ formData }) => {
+    if (formData.group_id === -1) {
+      delete formData.group_id;
+    }
+    const result = await dispatch(submitAllocation(formData));
+    if (result.status === "success") {
+      dispatch(showNotification("Allocation saved"));
+      dispatch(fetchAllocations());
+    }
+  };
+
+  const allocationFormSchema = {
+    type: "object",
+    properties: {
+      pi: {
+        type: "string",
+        title: "PI",
+      },
+      start_date: {
+        type: "string",
+        format: "datetime",
+        title: "Start Date",
+      },
+      end_date: {
+        type: "string",
+        format: "datetime",
+        title: "End Date",
+      },
```

Comment:
Is it really important to have the time as well as the date? If the allocations are nightly (and don't start/end at specific hours) then maybe it is easier not to have to fill out the exact times...

Reply:
Yeah ok it's probably best to leave it like this.